src/util/navigatePage.jsx

```diff
@@ -0,0 +1,9 @@
+import { useNavigate } from 'react-router-dom';
+
+export function useCustomNavigate() {
+    const navigate = useNavigate();
+
+    return (path) => {
+        navigate(path);
+    };
+}
```

Comment:
```suggestion
export function useCustomNavigate() {
    const navigate = useNavigate();
    const goto = (path) => navigate(path) 

    return { goto }
}
```

Access this on other files by doing this:
```jsx
const { goto } = useCustomNavigate()
```

Reply:
Done with this review